docs/contracts/events.rst

```diff
@@ -61,7 +62,12 @@ The hash of the signature of the event is one of the topics, except if you
 declared the event with the ``anonymous`` specifier. This means that it is
 not possible to filter for specific anonymous events by name, you can
 only filter by the contract address. The advantage of anonymous events
-is that they are cheaper to deploy and call.
+is that they are cheaper to deploy and call. It also allows you to declare
+four indexed arguments rather than three.
+
+.. note::
+    In the transaction log a non-anonymous event is indistinguishable from an anonymous one that
```

Comment:
Maybe rephrase to something like `Since the transaction log only stores the event data and not the type, you have to know the type of the event, including which parameter is indexed and if the event is anonymous or not in order to correctly interpret the data. In particular, it is possible to "fake" the signature of another event using an anonymous event.`